feat(changelog): load-balancers-deprecated-static-attachment-of-load 2023-09-19 (#2162)
scaleway/docs-content: changelog/september2023/2023-09-19-load-balancers-deprecated-static-attachment-of-load.mdx

---

> title: Static attachment of Load Balancer to Private Networks deprecated

Reviewer: ```suggestion
title: Static attachment is no longer available
```

status: deprecated
author:
    fullname: 'Join the #load-balancer channel on Slack.'
    url: 'https://scaleway-community.slack.com/archives/load-balancer'
date: 2023-09-19
category: network
product: load-balancers
---

> With the release of VPCs and their managed DHCP feature, we are reworking the way resources are attached to a Private Network, in order to offer a bullet-proof experience. To this end, we are removing the static attachment mode that allowed you to manually configure an IP address when attaching the Load Balancer to a Private Network from the console. This mode is not in line with the all-new Private Network management engine Scaleway is working on, and could result in non working configurations. In removing static attachment mode, we are encouraging adoption of best practices that will future-proof your architectures. Simultaneously, we are also working on set of new features that will allow you to manage your IP addressing space more granularly.

Reviewer: ```suggestion
With the release of [VPCs](/network/vpc/concepts/#vpc) and their managed [DHCP](/network/vpc/concepts/#dhcp) feature, we are reworking the way resources are attached to a Private Network, in order to offer a bullet-proof experience. To this end, the **static attachment mode** that allowed for manual configuration of an IP address when attaching a Load Balancer to a Private Network from the Scaleway console, is no longer available. This mode is not in line with the all-new Private Network management engine Scaleway is working on, and could result in non working configurations. In removing static attachment mode, we are encouraging adoption of best practices that will future-proof your architectures. Simultaneously, we are also working on a set of new features to manage your IP addressing space more granularly.
```

> Please note that while the feature will no longer be available from the console, it is still available in the API, though entering a deprecation cycle.

Reviewer: ```suggestion
While the feature will no longer be available from the Scaleway console, it is still available in the API, though entering a deprecation cycle.
```
